README.md

```diff
@@ -7,3 +7,38 @@ To learn more, please see https://vip.wordpress.com/documentation/vip-go/underst
 ## Usage
 
 All the directories here are required and will be available on production web servers. Any extra directories will not be available in production.
+
+## PHPCS
+
+This repo contains a starting point for installing and using a _local_ version of [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/) (PHPCS). To get started, you'll need [Composer](https://getcomposer.org/), then open a command line at this directory, and run:
+
+```sh
+composer install
```

Comment:
@GaryJones What are your thoughts on recommending using [`composer create-project automattic/vip-go-skeleton`](https://getcomposer.org/doc/03-cli.md#create-project) instead of cloning the repository? See this [Drupal project example](https://github.com/drupal-composer/drupal-project) as a reference.

Reply:
We actually create the repo when the site is created, so it's unlikely anyone would create the VIP Go repo directly.